v2.19.8

allelos opened the pull request:
Netdata dashboard version 2.19.8

ilyam8 commented on web/gui/dashboard/tv-react.html:
```diff
@@ -2,7 +2,7 @@
 <!-- SPDX-License-Identifier: GPL-3.0-or-later -->
 <html lang="en">
 <head>
-    <title>Netdata TV Dashboard</title>
```
@andrewm4894 👀

ilyam8 commented on web/gui/dashboard/tv-react.html:
```diff
@@ -2,7 +2,7 @@
 <html lang="en">
 <head>
-    <title>Netdata TV Dashboard</title>
+    <title>NetData TV Dashboard</title>
```
@andrewm4894 👀

andrewm4894 commented on web/gui/dashboard/tv-react.html:
```diff
@@ -2,7 +2,7 @@
 <html lang="en">
 <head>
-    <title>Netdata TV Dashboard</title>
+    <title>NetData TV Dashboard</title>
```
```suggestion
    <title>Netdata TV Dashboard</title>
```

andrewm4894 replied:
@allelos howdy - ok to use "Netdata" as opposed to "NetData"?

andrewm4894 commented on web/gui/dashboard/infographic.html:
```diff
@@ -3,7 +3,7 @@
 <head>
     <meta charset=utf-8>
-    <title>Netdata: Get control of your Linux Servers. Simple. Effective. Awesome.</title>
+    <title>NetData: Get control of your Linux Servers. Simple. Effective. Awesome.</title>
```
```suggestion
    <title>Netdata: Get control of your Linux Servers. Simple. Effective. Awesome.</title>
```

andrewm4894 commented on web/gui/dashboard/index-node-view.html:
```diff
@@ -2,7 +2,7 @@
 <html lang="en">
 <head>
-  <title>Netdata TV Dashboard</title>
+  <title>NetData TV Dashboard</title>
```
```suggestion
  <title>Netdata TV Dashboard</title>
```

andrewm4894 commented on web/gui/dashboard/demosites2.html:
```diff
@@ -2,7 +2,7 @@
 <html lang="en">
 <head>
-    <title>Netdata - Real-time performance monitoring, done right!</title>
+    <title>NetData - Real-time performance monitoring, done right!</title>
```
```suggestion
    <title>Netdata - Real-time performance monitoring, done right!</title>
```

andrewm4894 commented on web/gui/dashboard/demosites.html:
```diff
@@ -4,7 +4,7 @@
     <meta http-equiv="Refresh" content="0; url=https://www.netdata.cloud">
     <meta charset=utf-8>
-    <title>Netdata: Get control of your Linux Servers. Simple. Effective. Awesome.</title>
+    <title>NetData: Get control of your Linux Servers. Simple. Effective. Awesome.</title>
```
```suggestion
    <title>Netdata: Get control of your Linux Servers. Simple. Effective. Awesome.</title>
```

andrewm4894 commented on web/gui/dashboard/demo2.html:
```diff
@@ -2,7 +2,7 @@
 <html lang="en">
 <head>
-    <title>Netdata Dashboard</title>
+    <title>NetData Dashboard</title>
```
```suggestion
    <title>Netdata Dashboard</title>
```

andrewm4894 commented on web/gui/dashboard/demo.html:
```diff
@@ -2,7 +2,7 @@
 <html lang="en">
 <head>
-    <title>Netdata Dashboard</title>
+    <title>NetData Dashboard</title>
```
```suggestion
    <title>Netdata Dashboard</title>
```

andrewm4894 commented on web/gui/dashboard/dashboard.html:
```diff
@@ -518,8 +518,8 @@ <h1>Gauge.js</h1>
 <h1>Google Charts</h1>
-Netdata was originaly developed with Google Charts.
-Netdata is a complete Google Visualization API provider.
+NetData was originaly developed with Google Charts.
```
```suggestion
Netdata was originaly developed with Google Charts.
```

andrewm4894 commented on web/gui/dashboard/dashboard.html:
```diff
@@ -518,8 +518,8 @@ <h1>Gauge.js</h1>
-Netdata was originaly developed with Google Charts.
-Netdata is a complete Google Visualization API provider.
+NetData was originaly developed with Google Charts.
+NetData is a complete Google Visualization API provider.
```
```suggestion
Netdata is a complete Google Visualization API provider.
```

andrewm4894 commented on web/gui/dashboard/dashboard.html:
```diff
@@ -25,7 +25,7 @@
 <div class="container-fluid">
 
-<h1>Netdata Custom Dashboard <div data-netdata="system.cpu" data-chart-library="sparkline" data-height="30" data-after="-600" data-sparkline-linecolor="#888"></div></h1>
+<h1>NetData Custom Dashboard <div data-netdata="system.cpu" data-chart-library="sparkline" data-height="30" data-after="-600" data-sparkline-linecolor="#888"></div></h1>
```
```suggestion
<h1>Netdata Custom Dashboard <div data-netdata="system.cpu" data-chart-library="sparkline" data-height="30" data-after="-600" data-sparkline-linecolor="#888"></div></h1>
```

andrewm4894 commented on web/gui/dashboard/dashboard.html:
```diff
@@ -2,7 +2,7 @@
 <html lang="en">
 <head>
-    <title>Netdata Dashboard</title>
+    <title>NetData Dashboard</title>
```
```suggestion
    <title>Netdata Dashboard</title>
```

andrewm4894 commented on web/gui/dashboard/tv.html:
```diff
@@ -2,7 +2,7 @@
 <html lang="en">
 <head>
-    <title>Netdata TV Dashboard</title>
+    <title>NetData TV Dashboard</title>
```
```suggestion
    <title>Netdata TV Dashboard</title>
```

andrewm4894 commented:
I added a few suggestions to just use "Netdata" instead of the old "NetData" which i had fixed here: https://github.com/netdata/netdata/pull/11484

Just did'nt want to commit those small suggestions here myself but should be just small easy ones.

allelos commented:
> I added a few suggestions to just use "Netdata" instead of the old "NetData" which i had fixed here: #11484
> 
> Just did'nt want to commit those small suggestions here myself but should be just small easy ones.

@andrewm4894 There's no value in changing them here, since they will be overwritten with every dashboard release. These have to be changed on the dashboard repo.

ilyam8 commented:
> These have to be changed on the dashboard repo.

The PR is https://github.com/netdata/dashboard/pull/303

allelos commented:
I'm closing this since changes from 2.19.8 alongside with `NetData -> Netdata` will be on `2.19.9` release.
